apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -50,6 +50,29 @@ async function processPageAbout(page, api, { locale }) {
   return page;
 }
 
+async function processPageConsultation(page, api, { locale }) {
+  const { blocks } = page;
+  const consultation = blocks.filter((block) => block.slug === "consultations");
```

Comment:
Consultation will include at least 2 groups: documents and multimedia. We should name these blocks accordingly.